library/cwm/src/lib/cwm/replace_point.rb

```diff
@@ -20,7 +20,16 @@ def initialize(id: "_placeholder", widget: Empty.new("_initial_placeholder"))
 
     # @return [UITerm]
     def contents
-      ReplacePoint(Id(widget_id), @widget)
+      # Use for contents empty widget, because if @widget passed to constructor
+      # will be used then CWM itself will handle events for this initial widget.
+      # This is against replace point idea that replace point due to its dynamic
+      # content will handle events itself
+      ReplacePoint(Id(widget_id), Empty(Id("___cwm_rp_empty")))
```

Comment:
OK, but now I come back to the reason why I originally suggested `@widget` here:
- I don't like the id being `"___cwm_rp_empty"` here and `"_initial_placeholder"` in the constructor.
- I don't understand why use `Empty`  here and `Empty.new` in the constructor. Well, they are a Term and AbstractWidget respectively, but I see accidental complexity of dummy widgets that are going to be replaced anyway. Can we DRY this?